packages/api/src/domain/webhook.ts

```diff
@@ -16,4 +16,7 @@ export const mapiWebhookType = [
 ] as const;
 export type MAPIWebhookType = (typeof mapiWebhookType)[number];
 
-export type WebhookType = DAPIWebhookType | MAPIWebhookType;
+export const PingWebhookType = `${Product.ping}` as const;
+export type PingWebhookType = (typeof PingWebhookType)[number];
```

Comment:
Ping type added

Reply:
This construct only makes sense for an array (of literals).

Can just declare as

```ts
export type PingWebhookType = 'ping';
```